packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -482,12 +480,9 @@ export class GCFBootstrapper {
         return;
       }
 
-      /**
-       * Note: any logs written before resetting bindings may contain
-       * bindings from previous executions
-       */
-      logger.resetBindings();
```

Comment:
Can we somehow preserve this behavior and make these removals opt-in?

Then we can safely roll out this version without breaking anything.

Reply:
That's an option. I was envisioning releasing as a major version bump, then as you upgrade gcf-utils, you'd start grabbing the context logger in the dep upgrade PR.

Reply:
I think our deps updater script will blindly create a PR without knowing the change, so we should let the oncall know about this.

Reply:
Restoring the original (buggy) behavior, but keeps it from being a breaking behavior change. In the future, when we migrate loggers in bots, we can remove this reset logic.